examples/conversion/compare_hf_and_megatron/compare.py

```diff
@@ -203,7 +203,7 @@ def get_model_class(model_class_name: str = None, is_vl_model: bool = False):
         return AutoModelForCausalLM
 
 
-def is_vision_language_model(model_path: str) -> bool:
+def is_vision_language_model(model_path: str, trust_remote_code: bool = False) -> bool:
```

Comment:
in these and other cases, does it make sense to have bool | None, such that we always opt into trust_remote_code for supported model repos without extra user input?